docs/source/resources/index.rst

```diff
@@ -0,0 +1,92 @@
+Resources
+=========
+
+.. contents:: |toctitle|
+    :local:
```

Comment:
The generated website has a :local: index in the right panel. Do we need it in the page contents as well?

Reply:
Hmm. I see what you mean. I'm mixed about it. I'm very used to seeing the contents right there at the top of the page. For Dataverse we made a decision to always have it even on pages that don't have many subheadings (for consistency). I started adding these before we switched to the current theme, which is where the sidebar comes from. I could probably train myself to look at the sidebar.

This affects every page in the docs so a dedicated issue is probably best if we'd like to change it.

Reply:
Yeah let's figure this out separately, I've opened #283.